package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rsk-network-settings",
-  "version": "0.3.0-beta",
```

Comment:
Do we need to update the version? Only the readme changed

Reply:
True, Removed